content/architecture/index.md

```diff
@@ -1,22 +1,22 @@
----

```

Comment:
If it's ok for you I will do it later because I'm working on top of the same branch so I will have a ton of conflicts. I have a plan to move content docs folder. The reason why I didn't was to not make a lot of diff in git so it is easier to review.

Reply:
> The reason why I didn't was to not make a lot of diff in git so it is easier to review.

FYI: It's not a problem as GitHub deals with it easily and detect the move and print it nicely:
![Screen Shot 2022-08-11 at 09 49 03](https://user-images.githubusercontent.com/17568639/184087742-4aa46301-dd7b-4661-bf2d-ef4e54711b48.png)
and it's not sth that affects the review 👍